Ports/python3/package.sh

```diff
@@ -9,7 +9,11 @@ useconfigure="true"
 files="${PYTHON_ARCHIVE_URL} ${PYTHON_ARCHIVE}
 https://www.python.org/ftp/python/${version}/Python-${version}.tar.xz.asc Python-${version}.tar.xz.asc
 https://raw.githubusercontent.com/python/cpython/942dd9f3f77eef08fabddbd9fb883a866ad6d4cb/PC/pycon.ico launcher.ico"
-auth_type="sig"
+# NOTE: Re-enable this when updating to 3.10.0, the rc doesn't have an .asc file (the download above will 404 in the meantime, but that's ok)
+if [ "$1" == "showproperty" ]; then
+    # Workaround for lint-ports.py
+    auth_type="sig"
+fi
```

Comment:
Would it make sense to use SHA256 at least temporarily (or maybe even just in general)?

Reply:
Given we recently switched a bunch of ports over to sha256 that makes sense, done :+1: